Lib/fontTools/ttLib/tables/_c_m_a_p.py

```diff
@@ -444,11 +444,11 @@ def compile(self, ttFont):
 		nameMap = ttFont.getReverseGlyphMap()
 		lenCharCodes = len(charCodes)
```

Comment:
Looks like `lenCharCodes` is now unused.

Reply:
Thanks, fixed.